app/helpers/task_list_helper.rb

```diff
@@ -60,20 +60,12 @@ def proceeding_id(application, ccms_code)
   end
 
   def display_new_page?(legal_aid_application, task_name)
-    return false if Setting.opponent_organisations? && task_name.eql?(:opponent_name)
-    return false unless %i[children_application opponent_name].include?(task_name)
+    return false unless task_name.eql?(:children_application)
 
-    [
-      starting_children_application_for_first_time?(legal_aid_application, task_name),
-      starting_opponents_name_for_first_time?(legal_aid_application, task_name),
-    ].any?(true)
+    starting_children_application_for_first_time?(legal_aid_application, task_name)
```

Comment:
could be simplified further to just `starting_children_application_for_first_time?(legal_aid_application, task_name)` by the looks of it